### PR TITLE
Create EF Migration

### PR DESCRIPTION
**Pull request by BrunoMilitoFrancois**

### CODE REVIEW

File: ZPastel.Persistence/Configuration/OrderConfiguration.cs [hunk 1]
```diff
@@ -13,15 +13,14 @@ public void Configure(EntityTypeBuilder<Order> builder)
...
-            builder.Property(o => o.CreatedOn).HasColumnName("CreatedOn");
+            builder.Property(o => o.TotalPrice).HasColumnName("TotalPrice").HasColumnType("decimal(18,2)");
             builder.Property(o => o.CreatedByUsername).HasColumnName("CreatedByUsername");
             builder.Property(o => o.CreatedById).HasColumnName("CreatedById");
             builder.Property(o => o.CreatedOn).HasColumnName("CreatedOn");
             builder.Property(o => o.LastModifiedById).HasColumnName("LastModifiedById");
             builder.Property(o => o.LastModifiedOn).HasColumnName("LastModifiedOn");
 
-            builder.HasMany(o => o.OrderItems).WithOne().HasForeignKey(e => e.OrderId);
+            builder.HasMany(o => o.OrderItems).WithOne().HasForeignKey(e => e.OrderItemId).Metadata.DeleteBehavior = DeleteBehavior.Restrict;
```
Comment (joseluz) [inline ZPastel.Persistence/Configuration/OrderConfiguration.cs]: Create a delete scenario to cover this config

Comment (joseluz) [reply ZPastel.Persistence/Configuration/OrderConfiguration.cs]: When deleting an Order we should delete its items as well

Comment (joseluz) [reply ZPastel.Persistence/Configuration/OrderConfiguration.cs]: Validate the changes querying directly the in memory DBContext in test layer

Comment (BrunoMilitoFrancois) [reply ZPastel.Persistence/Configuration/OrderConfiguration.cs]: Created delete scenario
Validated changes

File: ZPastel.Test/OrderEndpointTests.cs [hunk 2]
```diff
@@ -397,5 +411,34 @@ public async Task CreateOrder_WithNegativeAndZeroQuantityInOrderItem_ResponseSta
...
+
+        [Fact]
+        public async Task DeleteOrder_WithOrderItems_ShouldDeleteTheOrderAndAllOrderItems()
+        {
+            var order = dataContext
+                .Set<Order>()
+                .Where(o => o.Id == 2)
+                .SingleOrDefault();
+
+            order.Should().NotBeNull();
```
Comment (joseluz) [inline ZPastel.Test/OrderEndpointTests.cs:425]: Please check also the Order Items are not null and there are any entry on it

Comment (BrunoMilitoFrancois) [reply ZPastel.Test/OrderEndpointTests.cs:425]: Added order item validation

**Pull request merged**